hooks/mutations/delegation/useAcceptReceivedRequestToBeDelegate.ts

```diff
@@ -5,16 +5,19 @@ import { setDelegator } from "web3";
 
 export function useAcceptReceivedRequestToBeDelegate() {
   const { address } = useUserContext();
-  const onError = useHandleError();
+  const { onError, clearErrors } = useHandleError();
   const queryClient = useQueryClient();
 
   const { mutate, isLoading } = useMutation(setDelegator, {
     onError,
     onSuccess: (_data, { delegatorAddress }) => {
```

Comment:
This pattern is added to all of the mutations — when the transaction succeeds, clear the errors for that context